platform/view/services/db/driver/sql/driver.go

```diff
@@ -116,6 +127,6 @@ func getOps(config driver.Config) (common.Opts, error) {
 
 func getTableName(prefix, name string) (table string, valid bool) {
 	table = fmt.Sprintf("%s_%s", prefix, name)
-	r := regexp.MustCompile("^[a-zA-Z_]+$")
+	r := regexp.MustCompile("^[a-zA-Z_0-9]+$")
```

Comment:
Numbers are allowed in SQL, but not at the start of the table name (unless you quote it). We can allow numbers if we need them, but maybe the regex should block strings that start with a number then.

Reply:
I have introduced a new function that generates UUID without digits: GenerateUUIDOnlyLetters